README.md

```diff
@@ -34,7 +34,7 @@
 
 ## Prerequisites
 
-Mollie API client requires Node 4.8.x or higher to be installed.
+Mollie API client requires Node 6.9.x or higher to be installed.
```

Comment:
Isn't changing the required Node version a breaking change?

Reply:
I've checked our metrics on the used user-agents since January 1st. The lowest version is Node 6.9.1 with one request. So let's change this without a major version bump.

Reply:
🎉